sklearn/gaussian_process/gpr.py

```diff
@@ -107,6 +108,10 @@ def optimizer(obj_func, initial_theta, bounds):
         given, it fixes the seed. Defaults to the global numpy random
         number generator.
 
+    n_jobs : int, default: 1
+        Number of CPU cores used during the 'n_restarts_optimizer' iterations.
+        If given a value of -1, all cores are used.
```

Comment:
Regarding the `optimizer`, should I add a warning in the docstring or do something else?